src/azure-cli/azure/cli/command_modules/appconfig/_validators.py

```diff
@@ -69,27 +69,31 @@ def validate_separator(namespace):
 def validate_import(namespace):
     source = namespace.source
     if source == 'file':
-        if namespace.path is None or namespace.format_ is None:
-            raise CLIError("usage error: --path PATH --format FORMAT")
+        if namespace.path is None:
+            raise RequiredArgumentMissingError("Please provide the '--path' argument.")
+        if namespace.format_ is None:
```

Comment:
Can you run through our code base and refactor the general CLI error to explicit one? Like InvalidArgumentValueError, RequiredArgumentMissingError, MutuallyExclusiveArgumentError etc. Resolve this comment if you want to do that in a separete pr as this is mainly for bug fix.

Reply:
I only changed the error types that this PR was touching. I'll update all other CLIErrors in a separate PR.